docs/design/autoware-architecture/map/map-requirements/vector-map-requirements-overview/index.md

```diff
@@ -0,0 +1,83 @@
+# Vector Map creation requirement specifications
+
+## Overview
+
+Autoware relies on high-definition point cloud maps and vector maps of the driving environment to perform various tasks such as localization, route planning, traffic light detection, and predicting the trajectories of pedestrians and other vehicles.
+
+A vector map contains highly accurate information about a road network, lane geometry, and traffic lights. It is required for route planning, traffic light detection, and predicting the trajectories of other vehicles and pedestrians.
+
+Vector Map uses [lanelet2_extension](https://github.com/autowarefoundation/autoware_lanelet2_extension/blob/main/autoware_lanelet2_extension/docs/lanelet2_format_extension.md), which is based on the [lanelet2](https://github.com/fzi-forschungszentrum-informatik/Lanelet2) format and extended for Autoware.
+
+The primitives (basic components) used in Vector Map are explained in [Web.Auto Docs - What is Lanelet2](https://docs.web.auto/en/user-manuals/vector-map-builder/introduction#what-is-lanelet2). The following **Vector Map creation requirement specifications** are written on the premise of these knowledge.
+
+This specification is a set of requirements for the creation of Vector Map(s) to ensure that Autoware drives safely and autonomously as intended by the user. It does not cover how to operate specific map creation tools.
+
+- [Vector Map Builder - how to use](https://docs.web.auto/user-manuals/vector-map-builder/how-to-use/edit-maps)
```

Comment:
I think it would be better to refer to this.

https://autowarefoundation.github.io/autoware-documentation/pr-581/how-to-guides/integrating-autoware/creating-maps/#creating-a-vector-map

Reply:
Thank you for your suggestion. I changed files and folders structure in https://github.com/autowarefoundation/autoware-documentation/pull/581/commits/69fe00e91bec9d2794ef9e7a1091442c8d1ed382 .